src/pages/budget.tsx

```diff
@@ -0,0 +1,202 @@
+import { WithI18nResult, withI18n } from 'modules/I18n';
+
+import Container from 'modules/Common/containers/Container';
+import Layout from 'modules/Common/containers/Layout';
+import { MDXRemote } from 'next-mdx-remote';
+import React from 'react';
+import { ResponsiveLine } from '@nivo/line';
+import { ResponsivePie } from '@nivo/pie';
+import TextContent from 'modules/Common/components/TextContent';
+import classNames from 'classnames';
+import dayjs from 'dayjs';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
+
+const graphLineTheme = {
+  textColor: 'var(--colorBlack600)',
+  fontSize: 12,
+  axis: {
+    domain: {
+      line: {
+        stroke: 'var(--colorBlack400)',
+        strokeWidth: 1,
+      },
+    },
+    ticks: {
+      line: {
+        stroke: 'var(--colorBlack200)',
+        strokeWidth: 1,
+      },
+    },
+  },
+  grid: {
+    line: {
+      stroke: 'var(--colorBlack200)',
+      strokeWidth: 1,
+    },
+  },
+};
+
+const graphPieColors = [
+  'var(--colorPrimary)',
+  'hsl(205,80%,65%)',
+  'hsl(205,60%,70%)',
+  'hsl(205,40%,75%)',
+];
+
+const graphLineColors = ['var(--colorPrimary)'];
+
+const expenses = {
+  '2020-06-01': 20164,
+  '2020-07-01': 21061,
+  '2020-08-01': 24755,
+  '2020-09-01': 16682,
+  '2020-10-01': 30325,
+  '2020-11-01': 21116,
+  '2020-12-01': 23322,
+  '2021-01-01': 3045,
+  '2021-02-01': 3045,
+  '2021-03-01': 11566,
+  '2021-04-01': 14430,
+  '2021-05-01': 12045,
+  '2021-06-01': 12483,
+  '2021-07-01': 10803,
+  '2021-08-01': 12390,
+  '2021-09-01': 14839,
+  '2021-10-01': 42231,
+  '2021-11-01': 32106,
+  '2021-12-01': 13784,
+  '2022-01-01': 914,
+  '2022-02-01': 10946,
+  '2022-03-01': 45202,
+  '2022-04-01': 25530,
+  '2022-05-01': 37457,
+  '2022-06-01': 30765,
+};
+
+const totalExpendituresData: any = [];
+let accumulatedExpenditures = 0;
+
+Object.entries(expenses).forEach(([month, expendituresThisMonth]) => {
+  accumulatedExpenditures += expendituresThisMonth;
+  totalExpendituresData.push({
+    x: dayjs(month).toDate(),
+    y: accumulatedExpenditures,
+  });
+});
+
+export default function BudgetPage({ mdxContent }: WithI18nResult) {
+  const { t } = useTranslation();
+  const router = useRouter();
+
+  const fundingSources = [
+    {
+      id:
+        router?.locale === 'fr'
+          ? '🇫🇷🏛 Ministère de l’Europe et des Affaires Étrangères'
+          : '🇫🇷🏛 French Ministry for Europe and Foreign Affairs ',
+      value: 369383,
+    },
+    {
+      id: router?.locale === 'fr' ? '🇪🇺🏛 France Relance' : '🇪🇺🏛 French Covid Recovery Fund',
```

Comment:
> Use the same layout in all cases.

```suggestion
      id: router?.locale === 'fr'
        ? '🇪🇺🏛 France Relance'
        : '🇪🇺🏛 French Covid Recovery Fund',
```

Reply:
it must be related to my Prettier configuration, I have to look at

Reply:
Fixed with https://github.com/ambanum/OpenTermsArchive.org/pull/154/commits/fddf77fd99f9ed5a561e098f8061050715784db0

Reply:
Thank you, this was indeed the right way to do this 🙂